aircraft/f-14b/Nasal/fox2.nas

```diff
@@ -3698,13 +3698,12 @@ var AIM = {
 	},
 	
 	notifyInFlight: func (lat,lon,alt,rdr,typeID,typ,unique,thrust,callsign, heading, pitch, speed, distinct=1) {
-		var msg = notifications.ArmamentInFlightNotification.new("mfly", unique, 2, 21+typeID);
+		var msg = notifications.ArmamentInFlightNotification.new("mfly", unique, distinct?2:3, 21+typeID);
```

Comment:
can you change the parameter and associated logic from "distinct" to "is_deleted" or "is_inflight" or "is_final" (or something similar) just for clarity.

Reply:
Changed it. :)